### PR TITLE
Profile view

### PR DESCRIPTION
**Pull request by Emmelie83**
A profile view with background picture and bg opacity.

### CODE REVIEW

File: frontend/src/router/index.js [hunk 1]
```diff
@@ -47,6 +49,11 @@ const router = createRouter({
                 window.location.href = 'http://localhost:9000/logout';
             },
         },
+        {
+            path: "/user",
+            name: "userProfile",
+            component: UserProfileView,
+        },
```
Comment (Frlundsten) [inline frontend/src/router/index.js]:
```suggestion
        {
            path: "/user",
            name: "userProfile",
            component: UserProfileView,
            beforeEnter: (to, from, next) => {
                if (localStorage.getItem('userData') !== null) {
                    next();
                } else {
                    alert('You need to sign in to see your profile')
                    next({ name: 'home' }); 
                }
            }
        },
```

**Review by Frlundsten — CHANGES_REQUESTED** [review]
Looks good,  we could add a guard to only allow users that are signed in to view the '/user' route. Since we save userData in localstorage we could check on that.

**Review by Frlundsten — APPROVED** [review]
Great! 🚀

**Review by DevelopedByDanny — APPROVED** [review]
Yes

**Comment by Mattan41** [comment]
logout button changes to login when viewing path /user. User is till logged in and can book seats to movies etc.

**Pull request merged**